engines/lucene-9.8.0/build.gradle

```diff
@@ -0,0 +1,28 @@
+plugins {
+    id 'java'
+}
+
+version '1.0-SNAPSHOT'
+
+sourceCompatibility = 17
+
+repositories {
+    mavenCentral()
+    maven {
+        url "https://dist.apache.org/repos/dist/dev/lucene/lucene-9.8.0-RC1-rev-d914b3722bd5b8ef31ccf7e8ddc638a87fd648db/lucene/maven/"
```

Comment:
This bit needs to be removed before merging the PR, in order to switch from the temporary Maven repo that is used for voting on the Lucene 9.8 release (https://lists.apache.org/thread/sg6n375511k6zf31g7qbv0fc4rsvjb89) to the actual Lucene 9.8 release.

Reply:
This is now removed.